roles/download/defaults/main.yml

```diff
@@ -91,6 +91,20 @@ fluentd_image_tag: "{{ fluentd_version }}"
 kibana_version: "v4.6.1"
 kibana_image_repo: "gcr.io/google_containers/kibana"
 kibana_image_tag: "{{ kibana_version }}"
+prometheus_operator_image_repo: "quay.io/coreos/prometheus-operator"
+prometheus_operator_image_tag: "v0.3.0"
+node_exporter_image_repo: "quay.io/prometheus/node-exporter"
```

Comment:
I really would like to see appname as a prefix for the var so we can easily tell what app it's for